static/js/hotkey.js

```diff
@@ -427,6 +451,10 @@ exports.process_hotkey = function (e, hotkey) {
     }
 
     if (hotkey.message_view_only && ui_state.home_tab_obscured()) {
+        // if up/down arrow key was pressed and streams menu is open
+        if ((event_name === 'up_arrow' || event_name === 'down_arrow') && subs.is_open) {
+            subs_arrow_keys(event_name);
+        }
```

Comment:
If I were to move this statement somewhere else, the code wouldn't be executed as up/down arrow keys are still set as `true` for the `hotkey.message_view_only` variable. 

Changing `hotkey.message_view_only` to false results in scrolling of the messages while the subscriptions menu is open.